### ISSUE TITLE
'import cvxportfolio as cvx' could create confusion

### ISSUE THREAD

**Issue by quant9** [cvxgrp/cvxportfolio#82]
`cvx` is used for `cvxpy` elsewhere in the code (and often in other repositories too) so wondering if the choice of `import cvxportfolio as cvx` could create confusion. Suggest potentially `cvp` or `cxp` (or the old `cp`).

Example: https://github.com/cvxgrp/cvxportfolio/blob/master/cvxportfolio/policies.py#L22

**Comment by enzbus** [comment]
Good point, just to be clear this affects only the examples, the user is free to choose whatever name when importing cvxportfolio. I used to `import cvxpy as cvx`, but cvxpy has changed that and now suggests `import cvxpy as cp`, so I thought of changing as well, and I guessed `cvx` would be a good monicker. Also, most cvxportfolio users would not interact with cvxpy directly, and power users would probably follow cvxpy naming convention. May I ask for @SteveDiamond 's opinion?
